src/textual_summary/generic_table_row.jsx

```diff
@@ -2,6 +2,8 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import IconOrImage from './icon_or_image';
 
+const filterValue = val => val == null ? '' : val;
```

Comment:
...`? '' : String(val)` and we could use `{filterValue(item.value)}` instead of `{\`${filterValue(item.value)}\`}`

Reply:
Will do. Thx!

Reply:
This can be further simplified into
```javascript
const filterValue = val => val || '';
```
No need to check for null, because `null == undefined` only if you'd wan to strictly check val to null or undefined.
```javascript
filterValue(false)
"" //output
filterValue([])
[]  //output
filterValue({})
{}  //output
filterValue(undefined)
""  //output
filterValue(null)
""  //output
```

Reply:
```js
`filterValue(0)`
"" // output
```

not really ;)

Reply:
Yeah that's correct, so if we want to print `false` and `0` in these places I am fine with it as it is now.